Core.Collectors/IO/AzureHelpers.cs

```diff
@@ -100,20 +94,20 @@ public static async Task<CloudQueue> GetStorageQueueCachedAsync(string queueName
             }
         }
 
-        public static async Task<CloudQueue> GetStorageQueueAsync(string queueName, string storageConnectionEnvironmentVariable = "AzureWebJobsStorage")
+        public static async Task<CloudQueue> GetStorageQueueAsync(string queueName, string storageAccountNameEnvironmentVariable = "StorageAccountName")
         {
-            CloudStorageAccount storageAccount = GetStorageAccount(storageConnectionEnvironmentVariable);
+            CloudStorageAccount storageAccount = await StorageAccountHelper.GetStorageAccountUsingMsi(storageAccountNameEnvironmentVariable, isQueue: true);
             CloudQueueClient queueClient = storageAccount.CreateCloudQueueClient();
             CloudQueue queue = queueClient.GetQueueReference(queueName);
             await queue.CreateIfNotExistsAsync().ConfigureAwait(false);
             return queue;
         }
 
```

Comment:
I thought that queue was not working with this version. Do I remember incorrectly?

Reply:
Queue uses managed identity. It is QueueTrigger that doesn't work (while binding it with Azure Functions during runtime) upon modifying conn string with managed identity. It requires Azure SDK and few other packages update. Will implement this as a new feature after receiving clarification from SDL review